src/cpp/core/r_util/RActiveSessions.cpp

```diff
@@ -14,7 +14,9 @@
  */
 
 #include <core/r_util/RActiveSessions.hpp>
-#include <core/r_util/RActiveSessionStorage.hpp>
+#include <core/r_util/RActiveSessionsStorage.hpp>
+
+#include <core/Log.hpp>
```

Comment:
(See the other comment about organising includes)
```suggestion

#include <core/Log.hpp>
#include <core/r_util/RActiveSessionsStorage.hpp>
```